.github/workflows/publish-release-artifact.yml

```diff
@@ -34,16 +35,23 @@ jobs:
             is_lts=false
           fi
 
-          echo "Version is $version, is_lts: $is_lts"
+          is_rc=false
+          if [[ ${version} == *"-SNAPSHOT" ]]; then
```

Comment:
Is this right?
https://github.com/jenkinsci/jenkins/releases/tag/jenkins-2.426.2-rc-1

Won't that show 2.426.2-rc34281.a_b... ?

Reply:
The mvn command executed L28 returns the project.version from the pom.xml, which doesn't correspond to the tag.

https://github.com/jenkinsci/jenkins/pull/8733/files#diff-11648584ad8169da42bec78a0209137863ad69726133730cefff2f1eb66d20c4R28

https://github.com/jenkinsci/jenkins/blob/78f1e9c8ebabab11d468f80572986e59a98a4d9c/pom.xml#L37-L77